pages/api/posting/delete.tsx

```diff
@@ -0,0 +1,61 @@
+import { firestore } from 'firebase-admin';
+import { NextApiRequest, NextApiResponse } from 'next';
+import initializeApi from '../../../lib/admin/init';
+import { userIsAuthorized } from '../../../lib/authorization/check-authorization';
+
+initializeApi();
+const db = firestore();
+const POSTINGS_COLLECTION = '/postings';
+
+interface PostingData {
+  postingId: string;
+}
+
+async function deletePosting(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
I think we'll need to have a way to make sure that only post author can delete posting.

Reply:
I added the changes, but wanted to let you know in case there are problems down the line with eslint, my branches are a bit messed up so I copied and pasted my files and committed them here through the github website instead of commiting and pushing from vs code.